assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/controllers.tf

```diff
@@ -13,6 +13,70 @@ resource "aws_route53_record" "etcds" {
   records = [aws_instance.controllers[count.index].private_ip]
 }
 
+# IAM Policy
+resource "aws_iam_instance_profile" "csi-driver-instance-profile" {
+  count = var.set_csi_driver_iam_role ? 1 : 0
+  name  = "${var.cluster_name}-iprof"
```

Comment:
Do we need the `-iprof` suffix? What for?

Reply:
To denote that it's an instance profile. Removed.